### PR TITLE
Added HMS integration to dev app

### PR DESCRIPTION
**Pull request by jkasten2**
This change is [Reviewable](https://reviewable.io/reviews/onesignal/onesignal-android-sdk/1078)

### CODE REVIEW

File: OneSignalSDK/app/build.gradle
Comment (mikechoch) [inline OneSignalSDK/app/build.gradle:72]: Should we also add in the location dependency?
`implementation 'com.huawei.hms:location:X.X.X.X'`

File: OneSignalSDK/app/src/main/java/com/onesignal/example/HmsMessageServiceAppLevel.java
Comment (Jeasmine) [inline OneSignalSDK/app/src/main/java/com/onesignal/example/HmsMessageServiceAppLevel.java:35]: is ok to be error lvl log?

**Comment by jkasten2** [comment]
Going to wait until after the major release to merge, since the dev is app has changed in the 4.0.0 branch.

**Review by emawby — APPROVED** [review]
Approving with new changes

**Pull request merged**
